main.cpp

```diff
@@ -18,13 +18,29 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+private: Rewriter &rewriter;
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
+    CastCallBack(Rewriter& _rewriter): rewriter(_rewriter) {}
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        const auto *styleCastExpr = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `styleCastExpr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
if (styleCastExpr != nullptr) {
    //body
}